src/components/Header/index.tsx

```diff
@@ -152,35 +164,29 @@ function Header() {
               Datasets
             </Link>
             <Link
-              href="/applications"
-              className="block px-4 py-2 hover:bg-hover-color hover:text-white"
+              href="/about"
+              className="block border-b border-gray-200 px-4 py-2 hover:bg-hover-color hover:text-white"
               onClick={closeMenu}
             >
-              <FontAwesomeIcon icon={faFileText} className="mr-2" />
-              Applications
+              <FontAwesomeIcon icon={faInfoCircle} className="mr-2" />
+              About
             </Link>
             <Link
-              href="/about"
-              className="block px-4 py-2 hover:bg-hover-color hover:text-white"
+              href="/requests"
+              className="border-200-gray block px-4 py-2 hover:bg-hover-color hover:text-white"
```

Comment:
not sure if this border is applied because border-200-gray is incorrect class. If you really want a border, can you instead use white-smoke color